Regression/WarpX-tests.ini

```diff
@@ -1044,8 +1044,8 @@ inputFile = Examples/Physics_applications/laser_acceleration/PICMI_inputs_1d.py
 runtime_params =
 customRunCmd = python3 PICMI_inputs_1d.py
 dim = 1
-addToCompileString = USE_PYTHON_MAIN=TRUE
-cmakeSetupOpts = -DWarpX_DIMS=1 -DWarpX_LIB=ON -DWarpX_APP=OFF
+addToCompileString = USE_PYTHON_MAIN=TRUE USE_OPENPMD=TRUE QED=FALSE
+cmakeSetupOpts = -DWarpX_DIMS=1 -DWarpX_APP=OFF -DWarpX_OPENPMD=ON -DWarpX_QED=OFF
```

Comment:
They need configuration commonality across the tests, so that they can use cached a cached build taking 30 s rather than a fresh build taking 10 minutes.

Reply:
Ok, sounds reasonable. Presumably at least one 1D test is using openpmd. But why QED=OFF? Is QED not supported in 1D?

Reply:
Some existing test was specifying that, and since none of the 1D tests use QED, that's how I got to commonality.

Probably not relevant to this, but QED won't compile for a mixed- (or single-, I think) precision build.